frontend/src/metabase/css/core/colors.module.css

```diff
@@ -67,6 +67,7 @@
   --mb-color-text-brand: var(--mb-base-color-brand-40);
   --mb-color-text-white: var(--mb-base-color-white);
   --mb-color-background: var(--mb-base-color-white);
+  --mb-color-background-info: var(--mb-base-color-gray-5);
```

Comment:
This is not exactly the same color as in the design, but we don't have color `#F0F0F0` anywhere in [the palette.](https://www.figma.com/design/YxiDXi1DzmXJCTMCuqXaaY/Applying-the-updated-colors?node-id=4001-16486&node-type=FRAME&t=Iwv4nAfH8QF1J5QG-0) This is the closet one that I could pick.